pandas/core/strings.py

```diff
@@ -1651,17 +1651,34 @@ def str_translate(arr, table, deletechars=None):
 
 def str_get(arr, i):
     """
+    Extract element for each component.
+
     Extract element from lists, tuples, or strings in each element in the
     Series/Index.
 
     Parameters
     ----------
     i : int
-        Integer index (location)
+        Integer index (location).
 
     Returns
     -------
     items : Series/Index of objects
+
+    Examples
+    --------
+    >>> x = pd.Series(["String", (1, 2, 4), ["a", "b", "c"]])
+    >>> x
+    0       String
+    1    (1, 2, 4)
+    2    [a, b, c]
+    dtype: object
```

Comment:
Can you add a numerical value, to show what happens.

Reply:
Since `get` is a method for strings, I didn't add a numerical value (using one returns 'NaN'). However, it can be added it if you say.